### PR TITLE
Beginner: Using static extensions

### PR DESCRIPTION
**Pull request by cambiata**
StringTools and Math examples of static ext use. 
Referring to Lambda. 
Write-your-own guide.

### CODE REVIEW

File: assets/content/cookbook/Beginner/using-static-extensions.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+using Math; // Adding 'Math' as a static extension
+
+class Main {
+  static function main() {
+    var f = 0.9;
+    trace(f.round()); // Works, because the float variable is "extended" with the round method!
+  }
+}
+```
+Another class often used for static extension is the **[Lambda](http://api.haxe.org/Lambda.html)**, wich extends iterable types (arrays and lists) with functional methods.
```
Comment (markknol) [inline assets/content/cookbook/Beginner/using-static-extensions.md]: Which

**Comment by markknol** [comment]
You're on a roll!! Nice!

**Comment by cambiata** [comment]
Thanks! Fixed typos, added *value* example in string section

**Comment by Simn** [comment]
codereview.haxe.org!

**Pull request merged**